pkg/resources/resource_source_load_generator.go

```diff
@@ -36,10 +36,12 @@ var sourceLoadgenSchema = map[string]*schema.Schema{
 	"comment":            CommentSchema(false),
 	"cluster_name":       ObjectClusterNameSchema("source"),
 	"size":               ObjectSizeSchema("source"),
+	"expose_progress":    IdentifierSchema("expose_progress", "The name of the progress subsource for the source. If this is not specified, the subsource will be named `<src_name>_progress`.", false),
```

Comment:
This passes correctly but based on the docs not figuring out what [identifier](https://materialize.com/docs/sql/create-source/load-generator/#monitoring-source-progress) it expects for the integration testing. Giving it unique names says it already exists in the catalog